index.js

```diff
@@ -116,6 +116,15 @@ function reset () {
   // Reset all changes in paths caused by addPath function
   modulePaths.forEach(function (path) {
     removePathHelper(path, require.main.paths)
+
+    // Delete from require.cache if the module has been required before.
+    // This is required for node >= 11
```

Comment:
@Kehrlann does this mean it will re-require the module each time it is been required?

Reply:
Only when calling reset(). When you reset, you don't want to find the aliased modules.

This fixes the tests on node 11 and 12.

Reply:
Oh, I see. Awesome! Approving